### PR TITLE
Home Page:

### PR DESCRIPTION
**Pull request by Gitang2108**
-added search box to home page
-added content in header footer
-added a new font

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
+import {MdAutocompleteModule} from '@angular/material';
+import{MdSelectModule} from '@angular/material';
 
 import { AppComponent } from './app.component';
 import { HeaderComponent } from './header/header.component';
 import { FooterComponent } from './footer/footer.component';
 import { SignupComponent } from './signup/signup.component';
+import { HomeComponent } from './home/home.component';
+
+
```
Comment (srishtihunjan) [inline src/app/app.module.ts]: can you remove this extra lines?

**Review by gdeepchadha — COMMENTED** [review]
Looks good.

File: src/app/home/home.component.html [hunk 2]
```diff
@@ -1,3 +1,24 @@
-<p>
-  home works!
-</p>
+
+
```
Comment (srishtihunjan) [inline src/app/home/home.component.html]: extra newline can be removed

**Review by gdeepchadha — DISMISSED** [review]

File: src/app/home/home.component.ts [hunk 3]
```diff
@@ -7,9 +9,27 @@ import { Component, OnInit } from '@angular/core';
...
+      'burgers',
+      'tacos',
+      'Pizza'
+     ];
+
+     location: FormControl = new FormControl();
+
+     locations=[
+      'Gainesville',
+      'Mumnai',
```
Comment (srishtihunjan) [inline src/app/home/home.component.ts]: typo - "Mumbai"? should be changed so that its easier for testing.

**Review by srishtihunjan — CHANGES_REQUESTED** [review]

**Pull request closed without merging**